Fishing Game/ViewControllers/FishingScreenViewController.swift

```diff
@@ -165,8 +171,8 @@ class FishingScreenViewController: UIViewController {
     
     // Updates the tackle box with additional cash
     func transitionToCatchScreen() {
-        Tacklebox.shared.cash += fishingDay.caughtFish.reduce(0) { $0 + Int($1.price) }
-        Tacklebox.save()
+        TackleboxService.shared.tacklebox.cash += fishingDay.caughtFish.reduce(0) { $0 + Int($1.price) }
```

Comment:
This is a lot of logic for the view controller to know about how the tackle box service should work. I would create a function inside the Tacklebox that does this